Makefile

```
@@ -138,4 +141,8 @@ test-e2e-single-node: install-go-junit-report
 	set -o pipefail; go test -tags=$(GOTAGS) -failfast -timeout 120m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e-single-node/ | ./hack/test-with-junit.sh $(@)
 
 bootstrap-e2e: install-go-junit-report install-setup-envtest
```

I'd suggest getting rid of this if you're going down the vendored route
```suggestion
bootstrap-e2e: install-go-junit-report
```